lua/lspconfig/ui/lspinfo.lua

```diff
@@ -264,12 +264,21 @@ return function()
     end
   end
 
+  local available_servers = util.available_servers()
```

Comment:
what is the purpose of storing this in a new local?

Reply:
forgot to restore it. I wanted to do something here before.